pkg/i2gw/providers/gce/converter_test.go

```diff
@@ -224,13 +224,12 @@ func Test_ToGateway(t *testing.T) {
 			expectedErrors: field.ErrorList{},
 		},
 		{
-			name: "no ingress class, default to gce ingress class behavior",
+			name: "empty ingress class, default to gce ingress class",
 			ingresses: map[types.NamespacedName]*networkingv1.Ingress{
 				{Namespace: testNamespace, Name: noIngClassIngressName}: {
 					ObjectMeta: metav1.ObjectMeta{
-						Name:        noIngClassIngressName,
-						Namespace:   testNamespace,
-						Annotations: map[string]string{networkingv1beta1.AnnotationIngressClass: gceIngressClass},
+						Name:      noIngClassIngressName,
+						Namespace: testNamespace,
```

Comment:
shouldn't we expect in this test to have an ingressclass annotation? 

Also, why not a ingressClass field?

Reply:
1.  GCE/GKE does not use ingressClass field. The annotation `kubernetes.io/ingress.class` is used to specify the ingress class.
2. This test case is regarding the scenario that neither annotation or ingressClass field is set. In such case, GKE default it to be an external ingress.

Both information can be found here https://cloud.google.com/kubernetes-engine/docs/how-to/load-balance-ingress#create-ingress